internal/access/access_test.go

```diff
@@ -33,7 +33,7 @@ func setupDB(t *testing.T) *gorm.DB {
 	err = data.CreateOrganization(db, org)
 	assert.NilError(t, err)
 
-	db.Statement.Context = context.WithValue(db.Statement.Context, "org", org)
+	db.Statement.Context = context.WithValue(db.Statement.Context, data.OrgCtxKey{}, org)
```

Comment:
The goal of the unique context key is to prevent other packages from being able to set the field.

So I think we'd also want a `WithOrg`, and `OrgFromContext` functions in `data` to hide this.